### PR TITLE
Refactor SCSS to remove duplication between facia and content bootstraps

### PR DESCRIPTION
**Pull request by johnduffell**
As per @OliverJAsh comment on https://github.com/guardian/frontend/pull/12289

We should have a common stylesheet between facia and content, as it's full of duplication anyway.

### CODE REVIEW

File: static/src/stylesheets/content.scss [hunk 1]
```diff
@@ -1,14 +1,13 @@
 @charset 'UTF-8';
 
+@import '_facia-content';
+
 /* ==========================================================================
    Variables and mixins
    ========================================================================== */
 
-@import '_vars';
```
Comment (sndrs) [inline static/src/stylesheets/content.scss]: i'm not sure about abstracting these, they're kind of 'settings' partials, it's not clear there's present now?

**Comment by sndrs** [comment]
we have a `_head.common.scss`, maybe call it `_common.scss`?

File: static/src/stylesheets/content.scss (hunk 1 above)
Comment (johnduffell) [inline static/src/stylesheets/content.scss]: will do, I didn't really know why things are there I just had a bash at it!

**Comment by johnduffell** [comment]
I just think the word common always causes problems as it isn't specific, it just says what it isn't rather than what it is.  So it turns into a dumping ground for anything no-one knows where they go (look at the common project in the scala for a prime example)  I prefer to say what things are in explicitly rather than just inviting anything.  Happy to discuss and change if necessary though.

**Comment by OliverJAsh** [comment]
Perhaps call it `facia-and-content` or `facia+content` to make it clear? I was confused at first!

**Comment by OliverJAsh** [comment]
Nice work! :+1:

**Comment by sndrs** [comment]
`_head.common.scss` currently is included in every `head.*.scss`, so that makes sense to me? this file will also be included in both main css files (i.e. there are none which will not have this). happy for any way of naming them to prevail, but i think they should be consistent?

**Comment by johnduffell** [comment]
on reflection since this is a file rather than a container that includes specifically named files (rather than being a container) we're not likely to get the dumping ground problem I mentioned

**Pull request merged**